ruby/puppet

```diff
@@ -1 +1 @@
-Subproject commit d93f59152eeb30c292b5e134d39967e03e41ff61
+Subproject commit f6f511d4046b05566e5495defa59832fe58a1fa4
```

Comment:
being as we're relying on some new functionality in Puppet, I'm assuming this was intentional?